### PR TITLE
Recipe App: Final Review

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

- Used controllers to handle requests and render empty views.
- Used layouts and templates for shared content.
- Use database migration files to maintain database schema.
- Use validations for models.
- Wrote SQL queries with ActiveRecord.
- Set up associations between models.
- Built a web app that requires the user to log in.
- Used Devise gem for authentication.
- Used integrity tests with RSPEC

## Important:

Use `db:seed` to populate database with sample data.

## Created by
@hunter4466 @smunozmo 

## Notes:

✅  - Used  Ruby on Rails.
✅  - Used  PostgreSQL.
✅  - Used  Rubocop.
✅  - No linters errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+   ```
+
+2. Install node dependencies
+
+   ```sh
+   npm install
+   ```
+
+3. Run the web server
+
```
Comment (GzimAsani) [inline README.md:44]: - [ ] Add on your readme file instructions on how to create/migrate your tables

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @smunozmo and @hunter4466                           
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by smunozmo** [comment]
Thank you! @GzimAsani

**Pull request merged**